### ISSUE TITLE
Implement Renggli et al

### ISSUE THREAD

**Issue by jack89roberts** [alan-turing-institute/ARC-LoCoMoSeT#4]
[Cedric Renggli, André Susano Pinto, Luka Rimanic, Joan Puigcerver, Carlos Riquelme, Ce Zhang, and
Mario Lučić. Which model to transfer? finding the needle in the growing haystack. In Proceedings of the
IEEE/CVF Conference on Computer Vision and Pattern Recognition, pages 9205–9214, 2022](https://openaccess.thecvf.com/content/CVPR2022/papers/Renggli_Which_Model_To_Transfer_Finding_the_Needle_in_the_Growing_CVPR_2022_paper.pdf)

**Task-aware strategy:**

- Train a logistic regression model on (a subset of) the new dataset, i.e. remove the last layer, do a forward pass to get features from the pre-trained model, train model on those features.

- Optionally: Same but with KNN classifier (but scales worse than above and performance was the same/slightly worse in the paper)

**Task-agnostic strategy: [NB: will save implementing this for #10]**

In this order:

1. Pick the model with the highest test accuracy on ImageNet.
2. Pick the model trained on the largest dataset.
3. Pick the model with the most parameters.
